### ISSUE TITLE
Missing "Mobile Security Decisions via Untrusted Input" module

### ISSUE THREAD

**Issue by Samuel-BF** [OWASP/SecurityShepherd#458]
After completing "content provider leakage" challenge, i encounter a 404 error on the next challenge.

And digging a bit, i find that this module is defined in [coreSchema.sql](https://github.com/OWASP/SecurityShepherd/blob/dev/src/main/resources/database/coreSchema.sql#L1552), but, unlike almost all other lessons, it has no corresponding jsp file in [lessons/](https://github.com/OWASP/SecurityShepherd/tree/dev/src/main/webapp/lessons) or in [challenges/](https://github.com/OWASP/SecurityShepherd/tree/dev/src/main/webapp/challenges) (and according to the debug log, there should be a corresponding file in lessons/).

In fact, the file was added in [feb 2016](https://github.com/SeanDuggan/SecurityShepherd/commit/ce43ea318bd9e81eb83b213e6ce1d3e0d9243021#diff-c034d2e1dce91db0b8d92b165aa0cd47) by @SeanDuggan . Why has it disappeared ? Is it only for me ?

Another module in in the same case is "Mobile Content Provider" (ID : a845ec1943a6342956a48cdc8ca60f40036b68a810109d0b9d2a35271377980 ).

**Version** : git dev.

**Comment by markdenihan** [comment]
@Samuel-BF Good Find. @SeanDuggan Any ideas here? Should the levels view's just be added again?

**Comment by SeanDuggan** [comment]
Yeah it looks that way, I’ll spin up a dev instance and take a look

**Comment by Samuel-BF** [comment]
Simply adding back the jsp file and the corresponding language file makes the level work again ("5e2b61c679d1f290d23308b3b66c3ec00cd069f1483b705d17f2795a4e77dcb6"). But maybe existed some other translations that also disappeared in the meantime, I don't know.

And I'm not already at the "Mobile Content Provider" level to test also this one.

**Comment by LargeCow** [comment]
Does anyone have a fix for this issue? Bummed I can't keep playing

**Comment by markdenihan** [comment]
An admin can disable the level in any version less than 3.1, and that will
allow you to continue with the levels. We'll release a patch very soon
